core/controller/src/main/scala/whisk/core/controller/Rules.scala

```diff
@@ -313,7 +314,7 @@ trait WhiskRulesApi extends WhiskCollectionAPI with ReferencedEntities {
             WhiskTrigger.put(entityStore, oldTrigger.removeRule(ruleName))
           }
 
-          val triggerLink = ReducedRule(actionName, Status.INACTIVE)
+          val triggerLink = ReducedRule(actionName, status)
```

Comment:
Perhaps we should add a test to enforce an inactive rule does not become active with update which changes the trigger.

active rule: update with new trigger, or update with new action -> rule stays active
inactive rule: update with new trigger, or update with new action -> rule stays inactive

Reply:
i could argue that updating the rule with a new trigger should enable it, matching create.

Reply:
The `create()` method is made in such a way that `active` is always returned for the rule status even if the rule is not actually in an active state. The updated test will fail because of this. If I used `wsk.rule.get()` instead of checking the `wsk.rule.create()` response, the tests would pass. Maybe we need to use `getStatus()` in the `create()` method.

Reply:
Referring to this line: https://github.com/dubeejw/openwhisk/blob/5814698cdff360e4d15e451545467982a2d4221f/core/controller/src/main/scala/whisk/core/controller/Rules.scala#L91.

Reply:
If creating a rule then it doesn’t exist so get status will not work. Did you mean something else?

Reply:
`create()` calls `update()` in the controller. We would only check the rule status during an update.

Reply:
@rabbah, fixed the tests. Can take care of the problem above when this is done https://github.com/apache/incubator-openwhisk/issues/3274.